portal-web/docroot/html/taglib/ui/search_iterator/list.jsp

```diff
@@ -37,6 +37,16 @@ if (fixedHeader) {
 %>
 
 <div class="table-responsive">
+	<c:if test="<%= searchContainer.isItemSelector() %>">
+		<div class="d-none" id="<%= namespace + id %>SelectMessage">
+			<%= LanguageUtil.get(request, "select-the") %>
+		</div>
+
+		<div class="d-none" id="<%= namespace + id %>StructureAndCloseTheModal">
+			<%= LanguageUtil.get(request, "structure-and-close-the-modal") %>
+		</div>
+	</c:if>
```

Comment:
Hey @pat270 , these language keys are too specific to the item selector, it really doesn't belong in the search_iterator. Instead of adding `isItemSelector` could we add `ariaLabel` as a property to `<liferay-ui:search-container-row>` so that we can pass the label fully formed from `view_item_selector_view_descriptor.jsp` and then add it to the row element?